config/locales/in_person_proofing/en.yml

```diff
@@ -134,11 +143,11 @@ en:
       address: Enter your current residential address
       barcode: Show this barcode and your state‑issued ID at a Post Office to finish
         verifying your identity
-      cta: Try verifying your ID in person
+      cta: Try verifying your identity at a Post Office
       id_address: Address on your ID
       po_search:
         location: Find a participating Post Office
-      prepare: Verify your identity in person
+      prepare: Verify your identity in person at a Post Office
       state_id_milestone_2: Enter the information on your state‑issued ID
```

Comment:
To Do: Look into in_person_proofing.headings.prepare inside app/javascript/packages/document-capture/components/document-capture.tsx -- it looks like a step indicator will also get updated because of this change- so will need to use a diff var if this is not okay - looking into this